cmd/krel/cmd/announce.go

```diff
@@ -0,0 +1,178 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"k8s.io/release/pkg/http"
+	"k8s.io/release/pkg/mail"
+	"k8s.io/release/pkg/release"
+	"k8s.io/release/pkg/util"
+)
+
+const (
+	sendgridAPIKeyFlag   = "sendgrid-api-key"
+	sendgridAPIKeyEnvKey = "SENDGRID_API_KEY"
+	nameFlag             = "name"
+	emailFlag            = "email"
+	tagFlag              = "tag"
+)
+
+// announceCmd represents the subcommand for `krel announce`
+var announceCmd = &cobra.Command{
+	Use:   "announce",
+	Short: "Announce Kubernetes releases",
+	Long: fmt.Sprintf(`krel announce
+
+krel announce can be used to announce already built Kubernetes releases to the
+%q Google group.
+
+If --nomock is specified, then the mail will be send to a test Google group
```

Comment:
maybe remove the `test` because it will send the email to the group right? if the `--nomock` is not set then will not send.

Reply:
The description was wrong/inversed. I made it more clear that nomock now sends to the actual mailing lists.